From d52c9f09cb27d8b8ba09670cddf18104d496b1e1 Mon Sep 17 00:00:00 2001
From: Eyal Reizer <eyalr@ti.com>
Date: Tue, 16 Oct 2012 11:24:56 +0300
Subject: [PATCH 19/23] board-da830-evm: disable high speed mode for SD
 interafce

Some of the Wilink8 wifi modules are showing sdio errors when enabling
the high speed capability of the mmc interface.
As we are working using 25Mhz sdio clock the high speed mode is not
needed and hence we disable it.

Signed-off-by: Eyal Reizer <eyalr@ti.com>
---
 arch/arm/mach-davinci/board-da830-evm.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm/mach-davinci/board-da830-evm.c b/arch/arm/mach-davinci/board-da830-evm.c
index f40e191..1d68b9c 100644
--- a/arch/arm/mach-davinci/board-da830-evm.c
+++ b/arch/arm/mach-davinci/board-da830-evm.c
@@ -680,8 +680,7 @@ static struct davinci_mmc_config da830_wl18xx_mmc_config = {
 	.wires		= 4,
 	.max_freq	= 25000000,
 	.caps		= MMC_CAP_4_BIT_DATA | MMC_CAP_NONREMOVABLE |
-			  MMC_CAP_POWER_OFF_CARD | MMC_CAP_MMC_HIGHSPEED |
-			  MMC_CAP_SD_HIGHSPEED,
+			  MMC_CAP_POWER_OFF_CARD,
 	.version	= MMC_CTLR_VERSION_2,
 };
 
-- 
1.7.9.5

